src/backend_model.cc

```diff
@@ -88,67 +83,115 @@ BackendModel::BackendModel(
   THROW_IF_BACKEND_MODEL_ERROR(
       TRITONBACKEND_BackendMemoryManager(backend, &triton_memory_manager_));
 
+  THROW_IF_BACKEND_MODEL_ERROR(ParseModelConfig());
+}
+
+TRITONSERVER_Error*
+BackendModel::ParseModelConfig()
+{
```

Comment:
From here to line 107 is a copy of lines 40-55 above. I think you forgot to delete the old code?